components/list/list-item-nav-mixin.js

```diff
@@ -30,11 +32,22 @@ export const ListItemNavMixin = superclass => class extends ListItemLinkMixin(su
 			.d2l-list-item-content ::slotted(*) {
 				width: 100%;
 			}
+			:host([current]) d2l-button-add,
+			:host([_has-current-parent]) [slot="add-top"] d2l-button-add,
+			:host([_next-sibling-current]) [slot="add"] d2l-button-add {
+				--d2l-button-add-line-color: var(--d2l-color-celestine);
+				--d2l-button-add-line-height: 3px;
+				--d2l-button-add-hover-focus-line-height: 3px;
+			}
 			:host([current]) [slot="outside-control-container"] {
 				background-color: var(--d2l-color-regolith);
 				border: 3px solid var(--d2l-color-celestine);
 				margin-block: 0;
 			}
+			:host([current][_show-add-button]) [slot="outside-control-container"] {
+				margin-bottom: -2px;
+				margin-top: -1px;
```

Comment:
This is needed so that the add button lines up properly with the current border. However it does cause some weirdness when hovering an adjacent item.